src/fsw/FCCode/DebugTask.cpp

```diff
@@ -20,12 +20,20 @@ void DebugTask::execute() {
 #ifndef FLIGHT
   start_cycle_f.set(false);
   if(auto_cycle_f.get()){
-    process_commands(_registry);
+    process_commands(_registry
+#ifdef DESKTOP
+    , false
+#endif
+    );
```

Comment:
```suggestion
    #ifdef DESKTOP
       const bool blocking = true;
    #else
       const bool blocking = false;
    #endif
    process_commands(_registry, blocking);
```

Just a little cleaner, IMO

Reply:
Yeah no good move for sure!

Reply:
Oh wait actually can't do this because the argument doesn't exist in HOOTL.

Reply:
Oh why not just let the argument exist and be useless in HITL...oh the compiler will complain?

Reply:
Whatever it doesn't matter

Reply:
Compiler errors for unused argument yeah.